OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs

```diff
@@ -131,7 +141,7 @@ public ActorSelectorLogic(Widget widget, World world, WorldRenderer worldRendere
 
 				var tooltipText = (tooltip == null ? "Type: " : tooltip.Name + "\nType: ") + a.Name;
 				allActorsTemp.Add(new ActorSelectorActor(a, editorData.Categories, searchTerms.ToArray(), tooltipText));
- 			}
+			}
```

Comment:
Why is this change here?

Reply:
Looks like he's removing a stray space, which should be fine (imho).

Reply:
Lets say its fine then :)